shapes.go

```diff
@@ -187,6 +187,39 @@ func (p *Proc) Triangle(x1, y1, x2, y2, x3, y3 float64) {
 	)
 }
 
+func (p *Proc) Bezier(x1, y1, x2, y2, x3, y3, x4, y4 float64) {
```

Comment:
please add some documentation to `Proc.Bezier` as well.
(it can be just copy-pasted from the `Bezier` func)

Reply:
fixed